Client/Frontend/Browser/OpenInHelper.swift

```diff
@@ -127,18 +138,18 @@ class OpenPdfInHelper: NSObject, OpenInHelper, UIDocumentInteractionControllerDe
     fileprivate var filepath: URL?
 
     required init?(response: URLResponse) {
-        guard let MIMEType = response.mimeType, MIMEType == MimeType.PDF.rawValue && UIApplication.shared.canOpenURL(URL(string: "itms-books:")!),
```

Comment:
There are a couple of things we could do here to make this even more perfect. 

1. Create a static var for the iBooks URL so it is clear that we aren't force unwrapping something dangerous. Something like `static var AppleiBooksURL = URL(string: "itms-books:")!` 
2.  Simply do a `let MIME = MimeType(rawValue: response.mimeType ?? "")` this will either fail to create a MimeType object or succeed and continue.